### PR TITLE
New v0.17.3 release

### PR DESCRIPTION
**Pull request by alvneiayu**
**Description of the change**

Update the Release Notes so we can trigger a new release `v0.17.3` including the improvements added at #653, #715, #726 and #727.

**Benefits**

New release.

**Possible drawbacks**

None

**Applicable issues**

N/A

**Additional information**

N/A

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ Latest release:
 
 [![](https://img.shields.io/github/release/bitnami-labs/sealed-secrets.svg)](https://github.com/bitnami-labs/sealed-secrets/releases/latest)
 
+## v0.17.3
+
+### Changelog
+
+- Unseal templates even when encryptedData is empty ([#653](https://github.com/bitnami-labs/sealed-secrets/pull/653))
+- Add new RBAC rules to make Sealed Secret compatible with K8s environments with RBAC enabled ([#715](https://github.com/bitnami-labs/sealed-secrets/pull/715))
+- Allow rencrypt/validate functionalities to work with named ports defined in the Sealed Secret service ([#726](https://github.com/bitnami-labs/sealed-secrets/pull/726)
```
Comment (juan131) [inline RELEASE-NOTES.md:13]: I'm missing an extra parenthesis here:
```suggestion
- Allow rencrypt/validate functionalities to work with named ports defined in the Sealed Secret service ([#726](https://github.com/bitnami-labs/sealed-secrets/pull/726))
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks! just a minor request

**Review by juan131 — APPROVED** [review]
LGTM

File: RELEASE-NOTES.md (hunk 1 above)
Comment (alvneiayu) [reply RELEASE-NOTES.md]: thanks @juan131, good catch. Fixed

**Review by agarcia-oss — APPROVED** [review]

**Pull request merged**